scipy/stats/_multivariate.py

```diff
@@ -629,24 +632,27 @@ def _cdf(self, x, mean, cov, maxpts, abseps, releps, lower_limit):
         # ensuring that lower bounds are indeed lower when passed, then
         # set signs of resulting CDF manually.
         b, a = np.broadcast_arrays(x, lower)
+        b, a = b - mean, a - mean  # _qmvn only accepts zero mean
         i_swap = b < a
         signs = (-1)**(i_swap.sum(axis=-1))  # odd # of swaps -> negative
         a, b = a.copy(), b.copy()
         a[i_swap], b[i_swap] = b[i_swap], a[i_swap]
         n = x.shape[-1]
         limits = np.concatenate((a, b), axis=-1)
 
-        # mvnun expects 1-d arguments, so process points sequentially
+        # qmvn expects 1-d arguments, so process points sequentially
+        # XXX: if cov.ndim == 2 and limits.ndim == 1, can avoid apply_along_axis
         def func1d(limits):
-            with MVN_LOCK:
-                return _mvn.mvnun(limits[:n], limits[n:], mean, cov,
-                                maxpts, abseps, releps)[0]
+            # res0 = _qmvn(maxpts, cov, limits[:n], limits[n:], rng)[0]
+            res = _qauto(_qmvn, cov, limits[:n], limits[n:],
```

Comment:
If it's desired, then `_qauto` could straightforwardly be modified to consider both `abseps` and `releps` to match the old behavior.

Reply:
Totally, it's straightforward. It's an "if" indeed though. I don't have a good feeling whether the full generality of `abseps` and `releps` is ever useful TBH.

Reply:
Probably not in this case. The main uses are where the box limits encompass the main body of the distribution, so the target value is within an order of magnitude near unity.